charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -1,6 +1,41 @@
 {
   "title": "datacatalog.json",
   "definitions": {
+    "CreateAssetRequest": {
+      "type": "object",
+      "required": [
+        "destinationCatalogID",
+        "resourceMetadata",
+        "details",
+        "credentials"
+      ],
+      "properties": {
```

Comment:
Add an optional property destinationAssetID. Katalog connector will use this field to generate a new asset (instead of test-assetID)

Reply:
There is a [assetID](https://github.com/rohithdv/fybrik/blob/7df26117650f25076205af6dbb6b4e7968341a15/charts/fybrik/files/taxonomy/datacatalog.json#L31)  field in CreateAssetResponse object. Does it not fulfil this purpose?

Reply:
No, it is in the response. The idea is to request a specific asset ID.

Reply:
Sorry I didnt understand. In `createAssetInfo` why would we be requesting a specific asset ? it is the job of `getAssetInfo` right ? Maybe I am missing something here.

Reply:
What assetID would the registration functionality provide? There are two options: either there is no control over the generated asset ID, or some meaningful name (or prefix) can be used.  For katalog-connector for example, you could use the original asset ID as a prefix for the new asset ID, thus after ingesting paysim-csv, we could get a new asset called paysim-csv-1234. @simanadler @roytman - what do you think?

Reply:
@shlomitk1 I have added an optional `destinationAssetID` in the interface. Please check. Thanks.